app/views/baskets/index.html.erb

```diff
@@ -1,2 +1,49 @@
-<h1>Baskets#index</h1>
-<p>Find me in app/views/baskets/index.html.erb</p>
+<% @cart.items.each do |item| %>
+<tr>
+<td>
+<%= image_tag item.pic , :alt => "#{item.title}" %>
+</td>
+<td>
+<%= link_to "#{item.title}" , store_path(item.product_id) %>
+</td>
+<td>
+<%= item.unit_price %>
+</td>
+<td>
+<%= item.quantity %>
+</td>
+<td>
+<%= item.total_price %>
+</td>
+<% end %>
+</tr>
+
```

Comment:
breaks 
 - no @cart passed to view from controller
 - no table or tbody html tags, breaking HTML semantic
 - no items in baskets